src/libs/actions/Session.js

```diff
@@ -234,6 +237,27 @@ function signIn(password, twoFactorAuthCode) {
         });
 }
 
+function signInWithShortLivedToken(accountID, email, shortLivedToken) {
+    Onyx.merge(ONYXKEYS.ACCOUNT, {...CONST.DEFAULT_ACCOUNT_DATA, loading: true});
+
+    createTemporaryLogin(shortLivedToken).then((response) => {
+        Onyx.merge(ONYXKEYS.SESSION, {
+            authToken: shortLivedToken,
+            accountID,
+            email,
+        });
+        if (response.jsonCode === 200) {
+            getUserDetails();
```

Comment:
Oh hmm why are we doing this? It should happen when we set the `authToken` no? 

https://github.com/Expensify/App/blob/1901df375c91eef883f455caf8b5dadeddb9bf9f/src/libs/Navigation/AppNavigator/AuthScreens.js#L185-L186